aio/tools/transforms/angular-base-package/processors/generateKeywords.js

```diff
@@ -120,27 +120,28 @@ function isString(value) {
 
 function tokenize(text, ignoreWords, dictionary) {
   // Split on whitespace and things that are likely to be HTML tags (this is not exhaustive but reduces the unwanted tokens that are indexed).
-  const rawTokens = text.split(/[\s\/]+|<\/?[a-z]+(?:\s+\w+(?:="[^"]+")?)*>/img);
+  const rawTokens = text.split(/[\s/]+|<\/?[a-z]+(?:\s+\w+(?:="[^"]+")?)*>/ig);
   const tokens = [];
-  for(let token of rawTokens) {
+  for (let token of rawTokens) {
     token = token.trim();
 
-    // Strip off unwanted trivial characters
-    token = token.replace(/^[_\-"'`({[<$*)}\]>.]+/, '').replace(/[_\-"'`({[<$*)}\]>.]+$/, '');
+    // Trim unwanted trivial characters from the start and end of the token
+    token = token.replace(/^[\s_"'`({[<$*)}\]>.,-]*([\w.-]*[^\W_])[\s_"'`({[<$*)}\]>.,-]*$/, '$1');
```

Comment:
FWIW, I find this regex a little difficult to read. Especially the `[^\W_]` part made my head spin 🤯

A couple of (optional) suggestions:
- Switch back to the two `replace()` calls (one for `/^.../` and one for `/...$/`).
- Replace `[^\W_]` with `[a-zA-Z0-9]`.
- Change the regex to: 
    ```suggestion
        token = token.replace(/^[\s_"'`({[<$*)}\]>.,-]*([\w.-]*?)[\s_"'`({[<$*)}\]>.,-]*$/, '$1');
    ```